source/lmp/pppm_dplr.h

```diff
@@ -28,6 +28,7 @@ class PPPMDPLR : public PPPM {
   ~PPPMDPLR() override {};
   void init() override;
   const std::vector<double> &get_fele() const { return fele; };
+  std::vector<double> *get_fele_pointer() { return &fele; }
```

Comment:
does getting the reference work for you ? 
```suggestion
  std::vector<double> &get_fele() { return fele; }
```